app/routes/new.tsx

```diff
@@ -9,20 +9,40 @@ import { requireUser } from '~/services/session.server';
 
 export const action = async ({ request }: ActionArgs) => {
   const user = await requireUser(request);
+  if (!user.id || !user.cellId) {
+    return json(
+      { errors: { body: 'User is missing, please contact support' } },
+      { status: 400 }
+    );
+  }
 
   const formData = await request.formData();
   const body = formData.get('body');
- const audience = formData.get('audience') as Audience
+  const audience = formData.get('audience') as Audience;
 
   if (typeof body !== 'string' || body.length === 0) {
-    return json({ errors: { body: 'Body is required' } }, { status: 400 });
+    return json(
+      { errors: { body: 'Por favor, digite a seu pedido de oração' } },
+      { status: 400 }
+    );
   }
 
-  if (audience !== Audience.CELL && audience !== Audience.ONLY_ME && audience !== Audience.CHURCH) {
-    return json({ errors: { body: 'Por favor, escolher somente Igreja, Célula ou Apenas eu' } }, { status: 400 });
-
+  if (
+    audience !== Audience.CELL &&
+    audience !== Audience.ONLY_ME &&
+    audience !== Audience.CHURCH
+  ) {
+    return json(
+      { errors: { body: 'Por favor, escolher somente Igreja, Célula ou Apenas eu' } },
+      { status: 400 }
+    );
```

Comment:
I looked into that but did not implement, cleaner but 1 line worth difference. If I had done my if statment in one line straight, it would be smaller than this allowedAudiences set up.